recipes-omnect/images/omnect-os-update-image.bb

```diff
@@ -11,15 +11,21 @@ LIC_FILES_CHKSUM = "\
 
 SRC_URI = "file://sw-description"
 
-COMPATIBLE_MACHINE = "rpi|phytec-imx8mm"
+COMPATIBLE_MACHINE = "rpi|phytec-imx8mm|omnect_grub"
```

Comment:
Think we already talked about the compatible machine mechanism. Should we now use omnect_uboot instead of rpi and phytec?

Reply:
we can not. i changed the override order in this PR that "rpi" and "phytec-imx8mm" will supersede "omnect_uboot".
rpi and phytec machine configs both set "omnect_uboot" and we use it in other occurrences. 
Using "omnect_uboot" here you could not distinguish between rpi and phytec. We could use "omnect_uboot" for the phytec devices, but it feels wrong. Imho we would need something like "omnect_uboot_imx" then.

Reply:
Nevermind you're right. For the variable `COMPATIBLE_MACHINE` we could do that. For the install/deploy steps and the sw-description not. I would prefer if we use it analog to the sw-description here.

Reply:
yes, that sounds reasonable.